liftoff_giftlist/src/main/resources/templates/createparty.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head"></head>
+<body>
+<nav th:replace="fragments :: nav-header"></nav>
+
+<div>
+    <form class="mb-3" method="post">
+        <label th:for="name" class="form-label">Group Name</label>
+        <input
+                type="text"
+                id="name"
+                class="form-control"
+                th:field="${party.name}"
+        />
+        <p class="error" th:errors="${party.name}"></p>
+        <label th:for="description" class="form-label">Group Description</label>
+        <input
+                type="text"
+                id="description"
+                class="form-control"
+                th:field="${party.description}"
+        />
```

Comment:
[general question] The way this is all set up, does the user who runs this createparty form automatically get assigned to be the party owner? Or is there another mechanism for that? And can that group owner change / be reassigned? 
These don't have to be figured out right now, but it's worth raising so we can address them.

Reply:
Yes. As of right now in the controller, when the form is submitted, the user who creates the Group is the assigned "Party Leader". If we have time we can certainly look into a way they can transfer that title / add another member to share that title to add additional people to the group.